hidden.php

```diff
@@ -2,31 +2,29 @@
 require_once 'functions.php';
 
 $db = connectDatabase();
-$pMin = 1;
-$pMax = 50000;
-$gMin = 1;
-$gMax = 5000;
-$rMin = 1;
-$rMax = 12;
+const min = 1;
+const pMax = 50000;
+const gMax = 5000;
+const rMax = 12;
+
 if (
-  isset($_POST['name']) && ($_POST['name'] !== '') &&
-  isset($_POST['points']) && ($_POST['points'] !== '') &&
-  isset($_POST['games']) && ($_POST['games'] !== '') &&
-  isset($_POST['rings']) && ($_POST['rings'] !== ''))
+  empty($_POST['name']) &&
```

Comment:
this is fine, but just for reference, instead of switching your if statement round you could have used `!empty()`